pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -98,6 +98,10 @@ type BundleTarget struct {
 	// data will be synced to.
 	ConfigMap *KeySelector `json:"configMap,omitempty"`
 
+	// Secret is the target Secret in Namespaces that all Bundle source
+	// data will be synced to.
```

Comment:
suggestion: I think it'd be helpful to make clear that this isn't going to work without being enabled.

```suggestion
	// Secret is the target Secret that all Bundle source data will be synced to. Using Secrets as targets is only supported if enabled at startup - by default, trust-manager has no permissions for writing to secrets and can only read secrets in the trust namespace.
```

Reply:
done